presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveDynamicPartitionPruning.java

```diff
@@ -252,8 +252,7 @@ public void testJoinWithImplicitCoercion()
         assertEquals(dynamicFiltersStats.getReplicatedDynamicFilters(), 0L);
         assertEquals(dynamicFiltersStats.getDynamicFiltersCompleted(), 1L);
         DynamicFilterDomainStats domainStats = getOnlyElement(dynamicFiltersStats.getDynamicFilterDomainStats());
-        // Dynamic filter is collected from build side but not used for such cases
-        assertEquals(domainStats.getSimplifiedDomain(), singleValue(BIGINT, 1L).toString(getSession().toConnectorSession()));
```

Comment:
It's still collected as `BIGINT`.

Reply:
right, `dynamicFilterDomainStats` doesn't know about the coerced value